pokedex/data/csv/pokemon_evolution.csv

```diff
@@ -396,10 +396,52 @@ id,evolved_species_id,evolution_trigger_id,trigger_item_id,minimum_level,gender_
 396,763,1,,,,,,,23,,,,,,,,,0,0
 397,768,1,,30,,,,,,,,,,,,,,0,0
 398,770,1,,42,,,,,,,,,,,,,,0,0
-399,773,1,,,,,,,,,220,,,,,,,0,0
+399,773,1,,,,,,,,,160,,,,,,,0,0
 400,783,1,,35,,,,,,,,,,,,,,0,0
 401,784,1,,45,,,,,,,,,,,,,,0,0
 402,790,1,,43,,,,,,,,,,,,,,0,0
 403,791,1,,53,,,,,,,,,,,,,,0,0
 404,792,1,,53,,,,,,,,,,,,,,0,0
 405,804,1,,,,,,,406,,,,,,,,,0,0
+406,462,3,83,,,,,,,,,,,,,,,0,0
+407,470,3,85,,,,,,,,,,,,,,,0,0
+408,471,3,885,,,,,,,,,,,,,,,0,0
+409,678,1,,25,1,,,,,,,,,,,,,0,0
+410,738,3,83,,,,,,,,,,,,,,,0,0
```

Comment:
magnezone, leafeon, glaceon, vikavolt now evolve using thunder/leaf/ice stones

Reply:
I missed the fourth line yesterday, that's female meowstic. Meowstic already has two rows in this file (ids 360 and 372), in the first one the `gender_id` field is empty, in the second one is set to `2` (male). I think we should either keep the first record (since there's no `evolved_forme_id` column in this table), or the other two.

Reply:
Done. This leaves a gap in the `id` column, but I don't think it's a problem (and row 325 was already missing)